performance/models.py

```diff
@@ -0,0 +1,111 @@
+    indicator_name = models.TextField()
+
+    # OTP stands for Office of the Premier
+    # national_comments is provincial only, headed "National Oversight Comments"
+    # dpme_comments is national only, headed "DPME Coordinator Comments"
+    # treasury_comments is national only, headed "National Treasury Coordinator comments
+
+    q1_target = models.TextField(blank=True)
+    q1_actual_output = models.TextField(blank=True)
+    q1_deviation_reason = models.TextField(blank=True)
+    q1_corrective_action = models.TextField(blank=True)
+    q1_national_comments = models.TextField(blank=True)
+    q1_otp_comments = models.TextField(blank=True)
+    q1_dpme_coordinator_comments = models.TextField(blank=True)
+    q1_treasury_comments = models.TextField(blank=True)
+
+    q2_target = models.TextField(blank=True)
+    q2_actual_output = models.TextField(blank=True)
+    q2_deviation_reason = models.TextField(blank=True)
+    q2_corrective_action = models.TextField(blank=True)
+    q2_national_comments = models.TextField(blank=True)
+    q2_otp_comments = models.TextField(blank=True)
+    q2_dpme_coordinator_comments = models.TextField(blank=True)
+    q2_treasury_comments = models.TextField(blank=True)
+
+    q3_target = models.TextField(blank=True)
+    q3_actual_output = models.TextField(blank=True)
+    q3_deviation_reason = models.TextField(blank=True)
+    q3_corrective_action = models.TextField(blank=True)
+    q3_national_comments = models.TextField(blank=True)
+    q3_otp_comments = models.TextField(blank=True)
+    q3_dpme_coordinator_comments = models.TextField(blank=True)
+    q3_treasury_comments = models.TextField(blank=True)
+
+    q4_target = models.TextField(blank=True)
+    q4_actual_output = models.TextField(blank=True)
+    q4_deviation_reason = models.TextField(blank=True)
+    q4_corrective_action = models.TextField(blank=True)
+    q4_national_comments = models.TextField(blank=True)
+    q4_otp_comments = models.TextField(blank=True)
+    q4_dpme_coordinator_comments = models.TextField(blank=True)
+    q4_treasury_comments = models.TextField(blank=True)
+
+    annual_target = models.TextField(blank=True)  # AnnualTarget_Summary2
+    annual_aggregate_output = models.TextField(blank=True)  # Preliminary_Summary2
+    annual_pre_audit_output = models.TextField(blank=True)  # PrelimaryAudited_Summary2
+    annual_deviation_reason = models.TextField(blank=True)
+    annual_corrective_action = models.TextField(blank=True)
+    annual_otp_comments = models.TextField(blank=True)
+    annual_national_comments = models.TextField(blank=True)
+    annual_dpme_coordincator_comments = models.TextField(blank=True)
+    annual_treasury_comments = models.TextField(blank=True)
+    annual_audited_output = models.TextField(blank=True)  # ValidatedAuditedSummary2
+
+    sector = models.TextField(blank=True)
+    programme_name = models.TextField(blank=True)
```

Comment:
```suggestion
    programme_name = models.TextField(blank=True)  # From SubProgramme in CSV export
```

please fix in the import code

Reply:
done